modules/adding-tab-pods-page.adoc

```diff
@@ -1,14 +1,48 @@
 // Module included in the following assemblies:
 //
-// * web_console/dynamic-plug-ins.adoc
+// * web_console/dynamic-plugin-example.adoc
 
 :_content-type: PROCEDURE
 [id="adding-tab-to-pods-page_{context}"]
 = Adding a tab to the pods page
-The following procedure adds a tab to the *Pod Details* page as an example extension to your plug-in.
+
+[NOTE]
+====
+The {product-title} web console runs in a container connected to the cluster you have logged into. See "Dynamic plug-in development" for information to test the plug-in before creating your own.
+====
 
 .Procedure
 
+. Visit the link:https://github.com/openshift/console-plugin-template[`console-plugin-template`] repository containing a template for creating plug-ins in a new tab.
+
+. Select *Use this Template* to create a GitHub repository.
+
+. Re-name the template with the name of your plug-in.
+
+. From your copied repository, clone it to your local machine so you can edit the code.
+
+. Edit the plug-in metadata in the `consolePlugin` declaration of `package.json`.
++
+[source,json]
+----
+"consolePlugin": {
+  "name": "my-plugin", <1>
+  "version": "0.0.1", <2>
+  "displayName": "My Plugin", <3>
+  "description": "Enjoy this shiny, new console plugin!", <4>
+  "exposedModules": {
+    "ExamplePage": "./components/ExamplePage"
+  },
+  "dependencies": {
+    "@console/pluginAPI": "*"
```

Comment:
Please [escape](https://docs.asciidoctor.org/asciidoc/latest/subs/prevent/) this asterisk.

Reply:
Does it need to be escaped? The preview looks fine.